### ISSUE TITLE
Support for Hybrid Connections in App Service

### ISSUE THREAD

**Issue by schmid37** [terraform-providers/terraform-provider-azurerm#7184]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Hybrid Connections are used to access endpoints and resources in an other network, as [described in the documentation here](https://docs.microsoft.com/en-us/azure/app-service/app-service-hybrid-connections).

Currently, azurerm_app_service does not support adding such a hybrid connection.
It is pretty annoying, adding the hybrid connection manualy after a deployment.

Hybrid connections could be created manualy in a seperate resource group and can be referenced by name and/without namespace (depends on implementation) in the app service. 

### New or Affected Resource(s)

azurerm_app_service

### Potential Terraform Configuration

```terraform
resource "azurerm_app_service" "webapp" {
  name                = "name"
  resource_group_name = "rg_name"
  location            = "West US 2"
  app_service_plan_id = azurerm_app_service_plan.sp.id

  .
  .
  .

  networking {
    hybrid_connection {
      name = "hybrid_connection_name_example"
      namespace = "namespace_example"
    }

    hybrid_connection {
      name = "hybrid_connection_name_example2"
      namespace = "namespace_example2"
    }
  }
}
```

**Comment by schmid37** [comment]
@tombuildsstuff will this be ever implemented?

**Comment by tombuildsstuff** [comment]
Fixed via #7224
